Mathlib/SetTheory/Ordinal/Veblen.lean

```diff
@@ -331,4 +340,131 @@ theorem veblen_eq_veblen_iff :
  veblenWith_eq_veblenWith_iff (isNormal_opow one_lt_omega0)
 
 end veblen
+
+/-! ### Epsilon function -/
+
+/-- The epsilon function enumerates the fixed points of `ω ^ ⬝`. -/
+scoped notation "ε_ " => veblen 1
```

Comment:
Shouldn't this be an `abbrev` instead, so that one can call it `.\eps_` outside of the `Ordinal` namespace?

Reply:
> I don't want to make `epsilon` a separate definition, since then it would necessitate writing API theorems for linking it back to `veblen`

Not if it's an `abbrev`

Reply:
Not much seems to break if I take your advice, so alright then!